content/doc/developer/publishing/requesting-hosting.adoc

```diff
@@ -62,12 +63,42 @@ Alternately, you can configure your plugin to be released automatically whenever
 In this case you only need write permission to GitHub; you never need local credentials.
 Follow link:/redirect/continuous-delivery-of-plugins[this guide] for details.
 
-== Categorize the Plugin
+=== Automate your GitHub repository
+
+==== Activate GitHub actions
+
+During the hosting process, you have included a shared workflow to generate changelogs automatically. Publishing changelogs as GitHub release notes is a good practice to keep your users informed about the changes in your plugin.
+
+To activate the workflow, you need to go to the `Actions` tab of your GitHub repository and click on the `I understand my workflows, go ahead and enable them` button.
+Follow the next section to create the necessary labels for your pull requests.
+
+==== Create the default labels
+
+Use the following script to create the default labels for your GitHub repository, used to categorize issues and pull requests and generate changelogs automatically:
+
+The script makes use of `gh`, you may need to install and link:[authenticate](https://cli.github.com/manual/gh_auth_login) first, if you haven't already.
+
+```sh
+https://gist.github.com/NotMyFault/b036718b73f1ad2a96faca315fc8ca68/raw
```

Comment:
An exhaustive list of labels used for release-drafter https://github.com/jenkinsci/.github/blob/master/.github/release-drafter.yml, with the default colors and descriptions.

Reply:
It seems like you want them to execute a shell command.  Doesn't that need to be something like:

```suggestion
curl -o labels https://gist.github.com/NotMyFault/b036718b73f1ad2a96faca315fc8ca68/raw
bash labels
```

Reply:
Thoughts about `&& bash...`, having it as a single command only then 👀

Reply:
Maybe store this file in RPU repo and link to the raw version instead of a gist in your personal account?

Also, maybe add a mention about reading the content first before executing blindly a content from Internet? (As an awareness precaution/advice)

Reply:
As there aren't that much lines in this script, I'm wondering if it wouldn't be better to list them in the doc and tell people to execute them themself.

Having them stored in a script in an official repository has the benefit of being a global reference, but it also give the not so good habit of executing a script from Internet.

I suppose it depends on whether this label list change frequently or not.

Reply:
> Maybe store this file in RPU repo and link to the raw version instead of a gist in your personal account?

Works either way. I thought of linking from the RPU to this part, but we can inline it for sure.

> I suppose it depends on whether this label list change frequently or not.

I don't think we have ever changed them for the release-drafter/cd action. I'm mostly using it when modernizing plugins because creating labels, with colors and descriptions always takes a good moment.